src/bootstrap/bootstrap.py

```diff
@@ -250,22 +246,15 @@ def default_build_triple(verbose):
             if verbose:
                 print("pre-installed rustc not detected: {}".format(e),
                     file=sys.stderr)
-                print("falling back to auto-detect", file=sys.stderr)
-
-    required = not platform_is_win32()
-    ostype = require(["uname", "-s"], exit=required)
-    cputype = require(['uname', '-m'], exit=required)
+                print("assuming x86_64-pc-windows-msvc", file=sys.stderr)
+            return 'x86_64-pc-windows-msvc'
```

Comment:
As far as I can tell, in the old code, if we were on Windows and didn't have a rustc toolchain, we'd always return this triple - either when uname failed, or further down in the `ostype` mapping, so we can just do that immediately here

Reply:
that's not true - it's possible to have `uname` installed in a Windows mingw environment.

Reply:
If `uname` returned, it still always returned this triple
```rs
   elif sys.platform == 'win32':
        # Some Windows platforms might have a `uname` command that returns a
        # non-standard string (e.g. gnuwin32 tools returns `windows32`). In
        # these cases, fall back to using sys.platform.
        return 'x86_64-pc-windows-msvc'
```

Reply:
Oh, my mistake, I missed the cases handling mingw and similar above that. I'll address that